sdk/python/tests/integration/offline_store/test_feature_logging.py

```diff
@@ -28,6 +28,11 @@
 @pytest.mark.integration
 @pytest.mark.universal_offline_stores
 @pytest.mark.parametrize("pass_as_path", [True, False], ids=lambda v: str(v))
+@pytest.mark.skipif(
```

Comment:
Does 2.0.3 also contain this bug? If so, even after snowflake is released, 3.8 tests will still fail once we remove `skipifs` as the latest pandas release in 3.8 is 2.0.3. Will we need to pin pandas version below 2.0.0 only for python 3.8 using environment markers?

Reply:
well actually the bug appears running [pandas testing](https://pandas.pydata.org/docs/reference/testing.html) only,
it does not affect to features though

Reply:
still, wdyt about adding something like this in our `ci` extras: `pandas < 2.0.0 : python_version == '3.8'`. Would be better than skipping the tests entirely.